babel/core.py

```diff
@@ -625,16 +625,39 @@ def currency_symbols(self) -> localedata.LocaleDataDict:
 
     @property
     def number_symbols(self) -> localedata.LocaleDataDict:
-        """Symbols used in number formatting.
+        """Symbols used in number formatting by number system.
 
         .. note:: The format of the value returned may change between
                   Babel versions.
 
-        >>> Locale('fr', 'FR').number_symbols['decimal']
+        >>> Locale('fr', 'FR').number_symbols["latn"]['decimal']
         u','
+        >>> Locale('fa', 'IR').number_symbols["arabext"]['decimal']
+        u'٫'
+        >>> Locale('fa', 'IR').number_symbols["latn"]['decimal']
+        u'.'
         """
         return self._data['number_symbols']
 
+    @property
+    def other_numbering_systems(self) -> localedata.LocaleDataDict:
+        """Mapping of othern numbering systems.
```

Comment:
```suggestion
        """Mapping of other numbering systems.
```

What does "other" mean here? Could we elucidate that some?

Reply:
Changed it now to:
```
Mapping of other numbering systems available for the locale.
See: https://www.unicode.org/reports/tr35/tr35-numbers.html#otherNumberingSystems